[Bug] Application Search empty results

**Describe the bug**

Application Search stopped working, cannot search for any application. Debugging shows powershell `Get-ChildItem : Could not find item` messages.

**To Reproduce**
1. Launch Ueli
2. Wait for scan to complete
3. Search for any application such as Notepad

**Expected behavior**
Application Search should return the applications, not empty results

**Screenshots**

```
[4/28/2026, 7:30:41 PM][INFO] Reading settings from C:\Users\Lucas\AppData\Roaming\ueli\ueli9.settings.json

[4/28/2026, 7:31:39 PM][ERROR] Failed to get search result items for extension with id 'ApplicationSearch.Reason: Get-ChildItem : Could not find a part of the path 'D:\Games\STAR WARS Battlefront II\ModData\Default\Update'.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ReadError: (D:\Games\STAR W...\Default\Update:String) [Get-ChildItem], DirectoryNotFound 
   Exception
    + FullyQualifiedErrorId : DirIOError,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \AssetTool.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\... \AssetTool.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \DataEditor.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...\DataEditor.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \MakeRes.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...e\ \MakeRes.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \PakInspector.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...akInspector.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \SF3ClientDebug.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...ClientDebug.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \SF3ClientFinal.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...ClientFinal.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \SF3LegacyLauncher.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...acyLauncher.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \SF3LevelEditor.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...LevelEditor.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
Get-ChildItem : Could not find item D:\Games\Steam\steamapps\common\SpellForceThree\ \SteamCLI.exe.
At C:\Users\Lucas\AppData\Roaming\ueli\PowershellUtility\009ff69a-188d-408f-a62a-218c1ca5a82d.ps1:10 char:9
+         Get-ChildItem -File -Path $FolderPaths -Recurse -Include $Fil ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Games\Steam\...\ \SteamCLI.exe:String) [Get-ChildItem], IOException
    + FullyQualifiedErrorId : ItemNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
'
```

<img width="891" height="588" alt="Image" src="https://github.com/user-attachments/assets/65790a61-b0fe-4de4-8a39-e4db720e083e" />


**Environment**
 - OS: Windows 10
 - Version: 10 22H2, Build 19045.7184
 - Ueli 9.28.0
 - Electron 41.3.0
 - Node.js 24.15.0
- V8 14.6.202.33-electron.0

**Additional context**
Similar to issue #1481

After investigating, the first path failure is a symlink. The others are standard file paths though

--- a/src/main/Extensions/ApplicationSearch/Windows/usePowershellScripts.ts
+++ b/src/main/Extensions/ApplicationSearch/Windows/usePowershellScripts.ts
@@ -5,7 +5,7 @@ const getWindowsAppsPowershellScript = `
             [string[]]$FileExtensions
         )
 
-        Get-ChildItem -File -Path $FolderPaths -Recurse -Include $FileExtensions | Select-Object -Property Name, FullName, Extension, BaseName | ConvertTo-Json -Compress
+        Get-ChildItem -File -Path $FolderPaths -Recurse -Include $FileExtensions -ErrorAction SilentlyContinue | Select-Object -Property Name, FullName, Extension, BaseName | ConvertTo-Json -Compress
     }`;
 
 const getWindowsStoreApps = `

